### ISSUE TITLE
[Automation] Energy Scale Test case

### ISSUE THREAD

**Issue by gkeishin** [openbmc/openbmc-test-automation#204]
Energy scale testing
	- Power Capping changes and verify the Power voltage changes

**Comment by sivassrr** [comment]
https://github.com/openbmc/openbmc-test-automation/issues/188  - Didnt find exact equivalent . May be I have created in RTC. @gkeishin could you add equivalent development story here

**Comment by sivassrr** [comment]
https://github.com/openbmc/openbmc/issues/1182  - Power Thermal related story.

**Comment by sivassrr** [comment]
https://github.com/openbmc/openbmc/issues/1213 - Currently Parking all Energy Scale related stories under this.

**Comment by sivassrr** [comment]
@saqibkh  I am parking all development story under this.

**Comment by sivassrr** [comment]
https://github.com/openbmc/openbmc/issues/1183 also related to this.

**Comment by sivassrr** [comment]
Based on discussion in global scrum, @gtmills will be working on automation part of it. 

- [ ] Need to have list test cases associate with energy scale / power / thermal management

- [ ]  Come up with list of test cases which can automate  / manual execution

- [ ]  Create stories for automation -   Automation need to be done.

**Comment by sivassrr** [comment]
Probable list of TCs:

- [ ] BMC Sensor Reporting – IPMI / REST / GUI
- [ ] Reading Temperature
- [ ] Reading Power
- [ ] Power limit change at power off and during runtime
- [ ] Activate / Deactivate power limit
- [ ] Powercapp enable / disable

**Comment by ssombar** [comment]
George Keishing:
witherspoon phosphor-settings-manager[1060]: Input parameter for PowerCap is invalid Input: 500
Watts is not in range:556..3389

Steven Sombar:
  Power limits used to be 500...3050
 
Matt Spinler:
 I recently made a change.  The min is now 556 watts.
 I did the fix with https://github.com/openbmc/openbmc/issues/2944
 
Matt Spinler:
  The old limits are 'output power', as seen by the OCC, and the 
  user only deals with input power, so the BMC code will take that value 
  they typed in and derate it (by .9) before sending it to the OCC.
  So I had to adjust the ranges of values that we allowed the user to type 
  in so that the OCC would still be happy with them. 
  500/.9 = 556,  3050/.9 = 3389
